docker-compose.yml

```diff
@@ -35,6 +35,7 @@ services:
       - "80:80"
     labels:
       io.balena.features.balena-socket: '1'
+      io.balena.features.balena-api: '1'
   dump1090-fa:
     build: ./dump1090-fa
     image: dump1090-fa
```

Comment:
one issue I have noticed is that the containers have `restart: always` for example here.

So the container get stopped by the curl but then gets restarted and this loop just continues. Maybe best to change to `restart: unless-stopped`

Reply:
Oops - should have commented on the line below this - https://github.com/ketilmo/balena-ads-b/blob/2a408fda4acdda05ad5d6eb82bb2e65e1f592ce0/docker-compose.yml#L42

Reply:
You can ignore this - `restart: always` does not seem to cause any issues here in fact

Reply:
Yes, if you call the stop-service from the API, it won't restart it.

Reply:
Yeah I realised that after I tested it 🤣🤦